### PR TITLE
Wait for sub counts to come in before redirecting

### PR DESCRIPTION
**Pull request by blowery**
…old start

Test live: https://calypso.live/?branch=alternate/wait-on-sub-counts-for-cold-start

### CODE REVIEW

File: client/reader/controller.js [hunk 1]
```diff
@@ -148,6 +151,38 @@ module.exports = {
...
+					defer( next );
+				}
+			} else if ( tries > -1 ) {
+				FeedSubscriptionStore.once( 'change', checkSubCount.bind( null, --tries ) );
+			} else {
+				defer( next );
+			}
+		}
+
+		checkSubCount( 3 );
```
Comment (bluefuton) [inline client/reader/controller.js]: Can we use `config.reader_cold_start_graduation_threshold` (+1) here?

**Comment by bluefuton** [comment]
Going to merge into #6382 and continue work there.

Thanks @blowery!

File: client/reader/controller.js (hunk 1 above)
Comment (blowery) [inline client/reader/controller.js]: oh, that's not the number of subs, it's the number of change events to consume before giving up. :)

Comment (bluefuton) [inline client/reader/controller.js]: Ah, it's for retries. Got it :)

**Pull request merged**